### PR TITLE
Source Airtable: update spec and doc

### PR DESCRIPTION
**Pull request by davydov-d**
## What
https://github.com/airbytehq/oncall/issues/1651

## How
Update spec.json and docs to state that the key should be **Personal Access Token**, not an **API Key**

### CODE REVIEW

**Review by erohmensing — COMMENTED** [review]
Took a look, but will wait until tests are passing to approve in case code changes. Also will pull in @bnchrch as he's coming on call

File: airbyte-integrations/connectors/source-airtable/source_airtable/spec.json
Comment (davydov-d) [reply airbyte-integrations/connectors/source-airtable/source_airtable/spec.json:61]: Tests passed successfully

**Review by erohmensing — APPROVED** [review]

**Pull request merged**
